### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by zoglo**
> ___Disclaimer___
> _No, this is not just some tutorial PR from some YouTuber to learn using GitHub, I wanted to test the current RC and went to the readme for quick instructions installing the demo :D._
> _Seen that it still doesn't have the LTS branch within the readme (5.2, not 5.3) and I kinda went all out after fixing that one thing_

### Description

- Updates the current working branch to 5.3 within the readme (was 5.2)
- Adds a table of contents
- Adds alternative texts for images
- Adds some badges
- Added code formatting to composer installation so we can copy those commands
- Added link to the current demo
- (Added .gitattributes so composer ignores the newly added picture within the /docs folder and it doesn't get downloaded into the `vendor` folder)

_I don't think we need to merge this downstream as 5.3 is the main branch presented on GitHub and packagist_

TL;DR:
**Dark mode readme**
image

**Light mode readme**
image

Live-example can be seen here:
https://github.com/zoglo/contao-demo/tree/update/readme

### CODE REVIEW

**Comment by aschempp** [comment]
I like! @sascha-mueller should we merge this?
